manual/guides/pullrequests.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: manual
+title: Pull Requests
```

Comment:
1) I think this file should be called pull_requests.md
2) I would expect a page titled "Pull Requests" to include both how to make PRs and how to review them. Possibly, it may be also helpful to add a couple of sentences on how to respond to PR review comments (e.g. how to deal with the suggesting functionality by committing the changes). 

If you'd like Reviewing a PR to be it's own page, that would be fine. If you'd like it to retain this title "Pull Requests," then the Making a Pull Request section should include links to existing guidance on how to make a PR. Guidance for making PRs can be found in my book and on the github help pages. Specific to this group, I would like to add that a PR in the group should:

- have GitHub labels
- have a GitHub project
- have a description in GitHub
- List requested reviewers (not everyone, but not no one. One or two PR reviewers is ideal. More can be the right choice sometimes. )
- be linked to an issue.
- be small (atomic, do just one thing)